tests/unit/nupic/engine/network_test.py

```diff
@@ -408,6 +410,15 @@ def testTwoRegionPyNodeNetwork(self):
     self.assertEqual(r2dims[1], 2)
 
 
+  def testGetRegion(self):
+    n = engine.Network()
+
+    region1 = n.addRegion("region1", "py.TestNode", "")
+
+    self.assertEqual(n.getRegion(TestNode), region1.getSelf())
```

Comment:
This should be `self.assertIs` since we want to make sure that it is the same region instance, not one that has the same values. Also, do we want it to match `region1.getSelf()` or `region1`?